app/helpers/application_helper.rb

```diff
@@ -40,6 +40,10 @@ def link_to_function(name, function, html_options = {})
     content_tag(:a, name, html_options.merge(:href => href, :onclick => onclick))
   end
 
+  def multiple_with_filter?
```

Comment:
Please leave a comment explaining why - as a quick scan will show this is duplicate code to what's in the controller